### ISSUE TITLE
Nice2Have: Connect a bare-metal node with FPGA to the instance (for OPTiMSoC)

### ISSUE THREAD

**Issue by oleg-nenashev** [librecores/librecores-ci#12]
TODO:
- [ ] - Define the scope
- [ ] - Define labeling policy and desired utilization restrictions
- ...

**Comment by oleg-nenashev** [comment]
@imphil is going to provide a machine, which we can use for a demo.

**Comment by oleg-nenashev** [comment]
@imphil Any updates regarding the bare-metal node? I'm stuck

**Comment by imphil** [comment]
@oleg-nenashev I guess I misunderstood you there, I'm sorry. I thought you'll have a docker container or something like that for me with the jenkins slave, and I'll install that then on the machine. Were you thinking of a baremetal node with SSH access?

**Comment by oleg-nenashev** [comment]
@imphil Yes. SSH Access is more or less secure out of the box, I would prefer to avoid dealing with JNLP encryption by now.

Docker image is not an option, because we need to connect physical interfaces like blasting and debugging cables. It can be done, but AFAIK it's pretty painful && no guarantee that drivers will operate correctly

**Comment by imphil** [comment]
ok, I see. I don't think SSH access will be a viable option for librecores CI in the longer term (I'll open a bug in a bit to outline what my thoughts are), let's first focus on the demo.

So what you need is:
- a local user for CI with SSH access
- this user must have access to the installed EDA tools

anything else?

**Comment by oleg-nenashev** [comment]
#3. There should be JDK 7 or 8 installed on the machine.
I think that's all by now
